CHANGES.md

```diff
@@ -1,5 +1,103 @@
 # ChangeList
 
+## = 1.12.1 (2021.12.09) =
```

Comment:
Wouldn't this be Dec 16 or Dec 17 (pre-release + a week or so)? Or are you planning to change this when doing the actual release?